Gopkg.toml

```diff
@@ -33,3 +33,12 @@ required = [
 [[constraint]]
   name = "github.com/percona/go-mysql"
   branch = "PMM-2.0"
+
+[[constraint]]
+  name = "github.com/percona/pmgo"
+  version = "^0.5.1" # We use exactly this version as it used by github.com/percona/percona-toolkit:3.0
```

Comment:
That actually means that we don't have to specify that constraint at all – it will be correctly inherited. Let's remove afterward.